src/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import Calculate from './components/calculator';
+import App from './App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <Calculate />
+    <App />
```

Comment:
[OPTIONAL] - I noticed that your `useEffect` Hook running twice when the quote from the API data is being fetched. 🤔

This is because outside of strict mode, React might run your hooks multiple times anyway, as it breaks the rendering phase up into pieces, and might pause or restart work. 👀

You can simply remove the `<React.StrictMode>` tags around the <App> tag, and this should disable strict mode for your app! 😎💪

```JavaScript
root.render(
    <App />
);
```